### PR TITLE
Add color-no-invalid-hex rule

### PR DESCRIPTION
**Pull request by martin91s**
## Suggested rule/changes(s):
* `color-no-invalid-hex` : `severity: (WARNING)`

## Reason for addition/amendment
Disallows invalid hex codes 

@netsells/frontend - Please review

### CODE REVIEW

**Review by rowbi-wan — APPROVED** [review]

File: rules/core/color-no-invalid-hex/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### 📍 block-no-empty
```
Comment (samturrell) [inline rules/core/color-no-invalid-hex/README.md:1]: Change rule name

**Review by samturrell — CHANGES_REQUESTED** [review]

File: rules/core/color-no-invalid-hex/README.md (hunk 1 above)
Comment (rowbi-wan) [reply rules/core/color-no-invalid-hex/README.md:1]: I really shouldn't approve PRs while not feeling well

Comment (samturrell) [reply rules/core/color-no-invalid-hex/README.md:1]: We could also definitely generate this

Comment (martin91s) [reply rules/core/color-no-invalid-hex/README.md:1]: ooopsy

**Review by samturrell — APPROVED** [review]

**Pull request merged**